CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -1,5 +1,30 @@
 # Changelog 2.1
 
+## 2.1.1 - 2025-02-11
+
+### Added
+
+- Add documentation for the `NodeLatencyMonitor` feature. ([#6561](https://github.com/antrea-io/antrea/pull/6561), [@antoninbas])
+
+### Changed
+
+- Close connection to IPFIX collector explicitly on Stop for `FlowAggregator`. ([#6635](https://github.com/antrea-io/antrea/pull/6635), [@antoninbas])
+- Run the `IPPool` webhook handler when `SecondaryNetwork` is enabled. ([#6691](https://github.com/antrea-io/antrea/pull/6691), [@luolanzone])
+
+### Fixed
+
+- Install OpenFlow entries by PortStatus to fix an Antrea Agent failure on Windows, rely on PortStatus messages from OVS to install OpenFlow entries for a Pod instead of relying on a fixed timeout which can be hard to configure correctly; the code will now wait "indefinitely" but a `NetworkNotReady` event will be added to the Pod if the port is not ready after 30s. ([#6763](https://github.com/antrea-io/antrea/pull/6763) [#6889](https://github.com/antrea-io/antrea/pull/6889), [@wenyingd])
```

Comment:
Remove `Install OpenFlow entries by PortStatus to fix an Antrea Agent failure`

Reply:
Updated.